packages/vuelidate/src/core.js

```diff
@@ -501,7 +501,7 @@ export function setValidations ({
       const s = unwrap(state)
       return s ? unwrap(s[key]) : undefined
     })
-    : isProxy(state) ? state : reactive(state || {})
+    : state
```

Comment:
This was useful if you want to watch the top level state, which I think we dont really need. 99% of people wont replace the entire state.